tests/integration_tests/api/system_backend/test_wrapping.py

```diff
@@ -31,3 +29,11 @@ def test_unwrap(self):
         logging.debug("unwrap_response: %s" % unwrap_response)
         self.assertIn(member="secret_id_accessor", container=unwrap_response["data"])
         self.assertIn(member="secret_id", container=unwrap_response["data"])
+
+    def test_wrap(self):
```

Comment:
Thank you for including a test! We're missing coverage for `payload=None`, maybe `@parameterized.expand()` can be used to generalize the test?
https://github.com/hvac/hvac/blob/main/tests/integration_tests/api/system_backend/test_health.py#L17

Reply:
I added `@parameterized.expand()`. The coverage should be at 100% now.

Reply:
great! will need to reformat again with `black` though

Reply:
oh. done